databuilder/snippets/ehrql.py

```diff
@@ -0,0 +1,41 @@
+# :snippet minimal-ehrql-import-patients
+from databuilder.tables import patients
+
+year_of_birth = patients.date_of_birth.year
+# :endsnippet
+
+# :snippet minimal-ehrql-import-dataset
+from databuilder.query_language import Dataset
+
+dataset = Dataset()
+# :endsnippet
+
+# :snippet minimal-ehrql-set-population
+dataset.set_population(year_of_birth >= 2000)
+# :endsnippet
+
+# :snippet minimal-ehrql-extract-year-of-birth
+dataset.year_of_birth = year_of_birth
+# :endsnippet
+
+# :snippet minimal-ehrql-extract-birth-year
+dataset.birth_year = year_of_birth
+# :endsnippet
+
+# :snippet minimal-ehrql-register
+from databuilder.definition import register
+
+register(dataset)
+# :endsnippet
+
+# :snippet minimal-ehrql
```

Comment:
Maybe this should be above the other snippets, as they are derived from it. Maybe the file should also mirror `docs/ehrql-tutorial.md` and be called `databuilder/snippets/ehrql-tutorial.py`?